Formula/wine.rb

```diff
@@ -9,21 +9,16 @@ class Wine < Formula
   homepage "https://www.winehq.org/"
 
   stable do
-    url "https://dl.winehq.org/wine/source/3.0/wine-3.0.4.tar.xz"
-    mirror "https://downloads.sourceforge.net/project/wine/Source/wine-3.0.4.tar.xz"
-    sha256 "d45a88edbe7db363e297cacedc94e66df3464504a01c2eccf04f518066a6fb0c"
+    url "https://dl.winehq.org/wine/source/4.0/wine-4.0.tar.xz"
+    mirror "https://downloads.sourceforge.net/project/wine/Source/wine-4.0.tar.xz"
+    sha256 "6736cdee95b2b8bb021ec0c19497ed8cad5ae2c8bfdb7ab5dc687ff92a480d4d"
 
     # Patch to fix screen-flickering issues. Still relevant on 3.0.
```

Comment:
The patch should be removed.

Reply:
I had removed this locally previously but even though it was marked as a won't fix / resolved a long time ago it was still present on 3.0, I could update the patch to work on 4.0 if that is more suitable? @fxcoudert @SMillerDev 
https://bugs.winehq.org/show_bug.cgi?id=34166
Either way I will find time to do whichever choice is made tonight ( PST )